router/router.go

```diff
@@ -520,6 +568,33 @@ func (worker *workerT) workerProcess() {
 	}
 }
 
+func (worker *workerT) getThrottlingCost() (cost int64) {
```

Comment:
Instead of calculating this cost everytime for each message, can't we calculate it once, when creating the throttler?
Each router of every destination type, already keeps a map with all its destination ids and their configuration.